docs/CHANGELOG.asciidoc

```diff
@@ -92,6 +93,17 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+
+[[release-notes-6.8.6]]
+== {kib} 6.8.6
+
+[float]
+[[bug-6.8.6]]
+=== Bug fix
+Maps::
+* Sanitizes attribution {pull}52309[#52309]
```

Comment:
This is a little vague.  Would "Sanitizes attribution for coordinate map" be correct?

Reply:
I think it's vague on purpose since this is a security fix...